core/admin/EE_Admin_Page.core.php

```diff
@@ -1887,141 +1889,32 @@ private function _add_global_feature_pointers()
      */
     public function load_global_scripts_styles()
     {
-        /** STYLES **/
         // add debugging styles
         if (WP_DEBUG) {
             add_action('admin_head', array($this, 'add_xdebug_style'));
         }
-        // register all styles
-        wp_register_style(
-            'espresso-ui-theme',
-            EE_GLOBAL_ASSETS_URL . 'css/espresso-ui-theme/jquery-ui-1.10.3.custom.min.css',
-            array(),
-            EVENT_ESPRESSO_VERSION
-        );
```

Comment:
extracted asset registration from `core/admin/EE_Admin_Page.core.php` and moved into `EspressoLegacyAdminAssetManager` and `LegacyAccountingAssetManager` for better SRP